GoogleSignIn/Sources/GIDSignIn_Private.h

```diff
@@ -44,6 +52,43 @@ typedef void (^GIDUserAuthCompletion)(GIDUserAuth *_Nullable userAuth, NSError *
 // @return NO if there is no user restored from the keychain.
 - (BOOL)restorePreviousSignInNoRefresh;
 
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
+
+// Starts an interactive consent flow on iOS to add scopes to the current user's grants.
```

Comment:
I don't think this comment will show for us. I am also noticing that we use this style for doc comments (e.g., `//` vs `///`) on internal/private APIs, but I wonder why. Don't we want these doc comments to what we expect them to (especially since we have things like `@param` in here)?

Reply:
We can discuss this convention in our weekly meetings.

Reply:
Done.